### PR TITLE
Task1: Counter

### PR DESCRIPTION
**Pull request by kristiyanpetkov**
I am ready with the first task!

### CODE REVIEW

File: src/com/clouway/threadcounter/ThreadCounter.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.clouway.threadcounter;
+
+/**
+ * Created by clouway on 15-10-15.
+ */
+public class ThreadCounter extends Thread {
+
+    public int i=0;
```
Comment (mgenov) [inline src/com/clouway/threadcounter/ThreadCounter.java]: Why "i" variable is with class level scope ?

**Comment by kristiyanpetkov** [comment]
ThreadCounter is not a proper name, so i will think for another proper name.

File: src/com/clouway/threadcounter/ThreadCounter.java (hunk 1 above)
Comment (kristiyanpetkov) [inline src/com/clouway/threadcounter/ThreadCounter.java]: I forgot to change it. Already fixed!

Comment (kristiyanpetkov) [inline src/com/clouway/threadcounter/ThreadCounter.java]: I use this variable in my Demo class.

**Comment by mgenov** [comment]
LGTM !

**Pull request merged**